SKILL.tsv

```diff
@@ -4038,8 +4038,8 @@ SKILL_20150428_004037	Key level +#{CaptionRatio}#
 SKILL_20150428_004038	Double Weapon Assault
 SKILL_20150428_004039	There is a chance of stealing enemy's item when you attack monsters while the flagged is pinned.
 SKILL_20150428_004040	{memo X}Reduce skill casting time. Applied once on the next skill.
-SKILL_20150428_004041	Damage decrease with more links{nl}Max. links #{CaptionRatio}#{nl}Duration #{CaptionTime}# seconds{nl}Use 16 Stamina
-SKILL_20150428_004042	Max. links #{CaptionRatio}#{nl}Duration #{CaptionTime}# seconds
+SKILL_20150428_004041	$Damage decreases depending on the amount of Links{nl}Maximum amount of Links: #{CaptionRatio}#{nl}Duration: #{CaptionTime}# seconds{nl}Consumes 16 Stamina
```

Comment:
I'd use "number" instead of "amount". Also: "Consumes 16 Stamina" -> "Stamina Cost: 16"